README.md

```diff
@@ -234,6 +234,23 @@ More details in [this blog post](http://blog.colinwilliams.name/blog/2015/01/26/
 I welcome pull requests for any test.check utility that seems halfway
 reasonable.
 
+## Development
+
+### Running tests
+
+For clj, be sure to use leiningen 2.5.2. (so it picks up .cljc files) and do as usual:
+
+    $ lein test
+
+For cljs [bensu/doo](https://github.com/bensu/doo) is configured so the tests can be run with:
+
+    $ lein doo node node-test
```

Comment:
Sent a [PR](https://github.com/bensu/doo/pull/27) to bensu/doo to clarify the minimum required node version in their README. Do you think we should add a note here, too?